torch/fx/node.py

```diff
@@ -421,7 +422,7 @@ def insert_arg(self, idx : int, arg : Argument) -> None:
 
         self._args = args_left + (arg,) + args_right
 
-        _new_input_nodes = {}
+        _new_input_nodes: Dict[Node, None] = {}
```

Comment:
Should the value be None or an Optional?

Reply:
I think it can never have a value of anything other than none, can it? The only thing that mutates that dict is the `setdefault` on the following line - which will set the value to `None` (it's really using the dict as a set of keys)